src/main/java/edu/cornell/kfs/fp/batch/service/impl/ProcurementCardSummaryFeedServiceImpl.java

```diff
@@ -185,29 +186,28 @@ protected void loadDataInDB(List<ProcurementCardSummaryEntry> entriesToLoad) {
 		for (ProcurementCardSummaryEntry entry : entriesToLoad) {
 			ProcurementCardSummaryEntry retrievedEntry = (ProcurementCardSummaryEntry) businessObjectService
 						.retrieve(entry);
-		//	entriesToLoad
-		//	.addAll(generateCalculatedSalaryFoundationTrackerCollection(psPositionJobExtractEntry));
 
-			
-			if (ObjectUtils.isNotNull(retrievedEntry)) {
-					entry.setVersionNumber(retrievedEntry
-							.getVersionNumber());
-				}
 			try {
 				businessObjectService.save(entry);
 			} catch (RuntimeException e) {
-			    logClassLoaderDebugInfo();
+			    logClassLoaderDebugInfo(entry, retrievedEntry);
 			    throw e;
 			}
 		}
 	}
 	
-    private void logClassLoaderDebugInfo() {
+    private void logClassLoaderDebugInfo(ProcurementCardSummaryEntry entry, ProcurementCardSummaryEntry retrievedEntry) {
+        LOG.info("logClassLoaderDebugInfo:: OJB Broker ClassHelper.ClassLoader.name: {}", org.apache.ojb.broker.util.ClassHelper.getClassLoader());
+        
         Class repoClass = org.apache.ojb.broker.metadata.ClassDescriptor.class;
-        LOG.info("Repository Class Loader: " + repoClass.getClassLoader());
-
+        LOG.info("logClassLoaderDebugInfo:: Repository Class Loader: {}", repoClass.getClassLoader());
+        
         Class pcardSummaryEntryClass = edu.cornell.kfs.fp.businessobject.ProcurementCardSummaryEntry.class;
-        LOG.info("ProcurementCardSummaryEntry Class Loader: " + pcardSummaryEntryClass.getClassLoader());
+        LOG.info("logClassLoaderDebugInfo:: ProcurementCardSummaryEntry Class Loader:{} ", pcardSummaryEntryClass.getClassLoader());

```

Comment:
For consistency with your other log messages, I would suggest adding a space between the colon and the placeholder.